scripts/lib-franklin.js

```diff
@@ -48,12 +50,19 @@ export function sampleRUM(checkpoint, data = {}) {
     if (window.hlx && window.hlx.rum && window.hlx.rum.isSelected) {
       const sendPing = (pdata = data) => {
         // eslint-disable-next-line object-curly-newline, max-len, no-use-before-define
-        const body = JSON.stringify({ weight, id, referer: window.location.href, generation: window.hlx.RUM_GENERATION, checkpoint, ...data });
+        const bdata = { weight, id, referer: window.location.href, generation: window.hlx.RUM_GENERATION, checkpoint, ...data };
         const url = `https://rum.hlx.page/.rum/${weight}`;
         // eslint-disable-next-line no-unused-expressions
-        navigator.sendBeacon(url, body);
+        navigator.sendBeacon(url, JSON.stringify(bdata));
         // eslint-disable-next-line no-console
         console.debug(`ping:${checkpoint}`, pdata);
+        sampleRUM.piggybacks
+          /* eslint-disable no-unused-vars */
+          .filter(([_, c]) => c === '*' || c === checkpoint)
```

Comment:
Please use a little more meaningful variable names, quite hard to figure out what _, c, u and t stands for.